### DISCUSSION TITLE
Segments to Image Sequences

### DISCUSSION THREAD

**Discussion by Jay123210599** [mifi/lossless-cut#2028] [Q&A]
How do I make image sequences out of segments I made while cutting videos? How do I make subtitles appear in them and the preview?

**Comment by mifi** [comment]
Right click on thesegment in the side panel to extract images from frames

you cannot make subtitles burned in. See faq

**Reply by Jay123210599** [reply]
What if I embed them, then? Will that work?

**Reply by mifi** [reply]
embed subtitles (burn in)? that's not possible
